suspicious-funding-ts/src/utils.ts

```diff
@@ -57,9 +57,9 @@ export const createFinding = (
   return Finding.fromObject({
     name: "Suspicious Funding Alert",
```

Comment:
Perhaps use "Malicious" here as well?

Reply:
That crossed my mind. I updated it, but now I'm wondering whether we should rename the bot. I think it _could_, but anything else we should keep in mind in regards to doing so?

I know for downstream purposes, the bot is referred to by its `botId`. But just checking with you.

Reply:
Hmm I don't think we really need to, unless it's specifically brought up to us